pkgs/by-name/cy/cyrus-imapd/cyrus-cli

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env sh
```

Comment:
So I gather this is the main output/executable of the package. Is this an entirely custom script, or is this sourced from upstream somewhere? It's not really clear.

Reply:
It's from debian cyrus-imapd package originally.

Reply:
> It's from debian cyrus-imapd package originally.

Hmmmmm... is this script present in packages for other distros and package managers? Having something Debian-specific in Nixpkgs sounds a bit weird to me

Reply:
> > It's from debian cyrus-imapd package originally.
> 
> Hmmmmm... is this script present in packages for other distros and package managers? Having something Debian-specific in Nixpkgs sounds a bit weird to me

Ah...
But this script is so useful that it could be there as the enter of cyrus-imapd management.
Upstream provides a lot of bin files, putting them all in the $PATH is not a good idea for nixos.